### ISSUE TITLE
[Screen Reader - Power Apps Admin - Select a template] Under Select a template blade, Narrator is not announcing the check/uncheck state of the checkbox on the table

### ISSUE THREAD

**Issue by sonimishra** [OfficeDev/office-ui-fabric-react#6637]
### Bug Report 
*User Experience:*
If, the Screen reader is not announcing the check/uncheck state of the checkbox available on the table, then the users who depends upon screen reader for navigation might get confuse and does not able to perform its function properly. 

MAS Ref:
*MAS 1.3.1 - Info and Relationships (40)*

 #### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)
 
#### Describe the issue:

*Pre-Requisite:*
1. Turn on narrator by pressing Ctrl + window + ENTER combination.

*Repro Steps:*
1. Navigate to https://admin.powerplatform.microsoft.com/ext/DataIntegration  and login with intusr.
2. Navigate to the Data Integration menu item on the left bar and select it.
3. Navigate to "+New Project" on the top right corner and select it.
4.New Data integration project screen with Connection tab item selected opens.
5. Navigate on the connection radio button and select it.
6. Navigate to Next button on bottom and select it.
7. Select a template screen appears. Navigate to controls of the screen.
8. Navigate with the narrator and reach first row of the table.
9. Verify whether the narrator is announce the check/unchecked state of checkbox when pressing Space bar.

#### Actual behavior: 
When user navigate with the narrator and reach first row of the table and press space bar to check and uncheck the checkbox then the narrator does not announce the state of the checkbox. Narrator announce 'space' key information.

#### Expected behavior:
When user navigate with the narrator and reach first row of the table and press spacebar for checking and unchecking the checkbox then the narrator should announce the state of the checkbox. 
 
*
![mas1 3 1-narrator 20is 20not 20announce 20check 20and 20unchecked 20state 20of 20checkbox png 20filename _mas1 3 1-narrator 20is 20not](https://user-images.githubusercontent.com/30456910/46719160-d4a5f800-cc8a-11e8-887f-5c12448a9b3b.png)

Note:*
1. This issue also exist with Chrome + JAWS.
2. Same issue is repro If we are activating “Keep all filters” .

**Comment by eveleens** [comment]
This is a dup of #6525

**Comment by cliffkoh** [comment]
I am able to repro this. Closing in favor of #6525. Thanks Eveleens.

**Comment by cliffkoh** [comment]
I added some investigation notes to #6525.
